.github/workflows/release.yml

```
@@ -0,0 +1,69 @@
+
+      # Build the package
+      - name: Prebuild
+        run: yarn workspace @coinbase/wallet-sdk prebuild
+
+      - name: Build Packages
+        shell: bash
+        run: yarn workspace @coinbase/wallet-sdk build
+
+      # Create a pull request to update the version
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v4
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)